src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -226,48 +226,50 @@ public void missingSoundMapping(RegistryEvent.MissingMappings<SoundEvent> event)
     }
 
     // done in an event instead of deferred to only enable if a mod requests it
-    public void registerFluids(RegistryEvent.Register<Fluid> event)
+    public void registerFluids(RegisterEvent event)
```

Comment:
Should these three `register*` methods be kept separate, or consolidated into a single method?

Reply:
I think keeping them separate is fine.